cmd/web/config/config_test.go

```diff
@@ -212,7 +212,22 @@ func TestValidatorType_Set(t *testing.T) {
 		args    args
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
```

Comment:
Excellent start! I think the test can be improved by verifying that the `.Set()` actually did the thing we want it to do, besides not generating an error.

This means you'll want to expand the test-case, which is of course absolutely OK (these skeleton test-cases aren't complete, they are just auto-generated starting points) Feel free to add fields and other checks in each test.

If I may make a recommendation, add a field `want` and make sure that `vt == want` in the test body.

E.g.:
```go
	tests := []struct {
		name    string
		vt      ValidatorType
    want    ValidatorType // To add
		args    args
		wantErr bool
	}
```

The test:
```go
			if err := tt.vt.Set(tt.args.v); (err != nil) != tt.wantErr {
						t.Errorf("Set() error = %v, wantErr %v", err, tt.wantErr)
			}

			if tt.vt != tt.want {
						t.Errorf("Expected Set(%q) to result in vt == %v", tt.args.v, tt.vt)
			}
```